setup.py

```diff
@@ -791,6 +792,11 @@ def run(self):  # noqa C901
             modpath="executorch.extension.training.pybindings._training_lib",
             dependent_cmake_flags=["EXECUTORCH_BUILD_EXTENSION_TRAINING"],
         ),
+        BuiltExtension(
+            src="extension/llm/runner/python_bindings*",  # @lint-ignore https://github.com/pytorch/executorch/blob/cb3eba0d7f630bc8cec0a9cc1df8ae2f17af3f7a/scripts/lint_xrefs.sh
```

Comment:
1. your path is incorrect, see comment above about install location
2. your output filename is incorrect, it should be something like `runner.so` named after your pybind11 module name.